crates/typos-dict/assets/words.csv

```diff
@@ -139,7 +139,6 @@ abotu,about
 abou,about,abound
 abount,about
 abourt,abort,about
-abouta,about
```

Comment:
Alternatives
- We report these as bad without suggesting a correction
  - We support this elsewhere though we might not support this here.  So could be easy, could be hard.
- We do a best-effort on joining the words
  - This would work in `abouta_foo` but not `abouta foo`
  - This would be a lot more work

Thoughts?

Reply:
Yes I did already think about reporting these as bad without suggesting a correction.
How about allowing corrections to contain spaces but if they do just not applying the corrections automatically?

So we could have e.g. `abouta,about,about a`, which would become:

```
error: `abouta` isn't a word, did you mean `about` or `about a`?
```

Reply:
> How about allowing corrections to contain spaces but if they do just not applying the corrections automatically?

Thats much more invasive and I'd liek to hold off for now.  Something like that should be discussed in an issue first so we can plan out different aspects before focusing on the implementation

Reply:
But I think we can address this in a followup PR. Not suggesting corrections that are likely to be wrong is still better than missing a potential typo since false automatic corrections are much more undesirable than just false negatives.

Reply:
I just opened #795 for tracking that. Are you alright with removing these corrections for now?

Reply:
With a small change, we now support disallowed words in the dictionary.  We should at least do that within this PR rather than completely drop them

Reply:
Oh yeah thanks, that does make much sense and is a very elegant solution.